libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1508,9 +1510,8 @@ string YulUtilFunctions::copyArrayToStorageFunction(ArrayType const& _fromType,
 			function <functionName>(slot, value<?isFromDynamicCalldata>, len</isFromDynamicCalldata>) {
 				<?fromStorage> if eq(slot, value) { leave } </fromStorage>
 				let length := <arrayLength>(value<?isFromDynamicCalldata>, len</isFromDynamicCalldata>)
-				<?isToDynamic>
-					<resizeArray>(slot, length)
-				</isToDynamic>
```

Comment:
`isToDynamic` is now unused. You should remove it from the parameter list below.

Reply:
Done